pkg/api/handlers_data.go

```diff
@@ -102,7 +102,7 @@ func handleWorkResponse(c *gin.Context, responseCh chan []byte) {
 			c.JSON(http.StatusOK, result)
 			return
 		// teslashibe: adjust to timeout after 10 seconds for performance testing
-		case <-time.After(10 * time.Second):
+		case <-time.After(15 * time.Second):
```

Comment:
just a small note as I'm walking this: maybe in the long run we might want to have a general `TIMEOUT` env var where we can tie these in

Reply:
Yeah I agree this makes sense. I feel like all configs should end up rolling into viper at the end

Reply:
I have created the following issue to address this:
https://github.com/masa-finance/masa-oracle/issues/497